runtime/compiler/optimizer/EstimateCodeSize.hpp

```diff
@@ -38,6 +38,7 @@ struct TR_CallTarget;
 
 class TR_EstimateCodeSize
    {
+   friend class InterpreterEmulator;
```

Comment:
what does the friendship enable / why is it needed?

Reply:
This is no longer needed after modifications. It was needed for 2 reasons in the previous version. First, for property flags of bytecodes which is moved to InterpreterEmulator now. Second, the `isInlineable` query which is made public now.